packages/fuels-accounts/src/predicate.rs

```diff
@@ -40,10 +40,17 @@ impl Predicate {
     }
 
     pub fn load_from(file_path: &str) -> Result<Self> {
-        let code = fs::read(file_path).map_err(|e| {
+        let clean_file_path = path_clean::clean(file_path);
+        let absolute_file_path = path::absolute(&clean_file_path).map_err(|e| {
```

Comment:
Not that we use `path::clean` would it make sense to use `canolicalize` instead of `absolute`? If we stick with `absolute` I would change the error messages because we currently say: `failed to canonicalize path`